doc/4-instrument/c64.md

```diff
@@ -37,12 +35,26 @@ the C64 instrument editor consists of two tabs: "C64" to control various paramet
 
 - **Volume**: volume sequence.
   - warning: volume sequence is global! this means it controls the chip's volume and therefore affects all channels.
-  - this macro becomes **Cutoff** when the **Volume Macro is Cutoff Macro** option is enabled in the C64 tab.
 - **Arpeggio**: pitch sequence.
 - **Duty**: pulse width sequence.
 - **Waveform**: select the waveform used by instrument.
 - **Pitch**: fine pitch.
+- **Cutoff**: filter cutoff.
 - **Filter mode**: select the filter mode.
 - **Resonance**: filter resonance sequence.
-- **Special**: ring and oscillator sync selector.
-- **Test/Gate**: when on, the test bit is set, which mutes the channel.
+- **Special**: ring and oscillator sync selector, as well as:
+  - **gate bit**:
+    - set (1): key on. if previous state was 0 it triggers envelope start/restart; if previous state was 1, it does nothing.
+    - reset (0): key off. if previous state was 1 it triggers envelope release; if previous state was 0, it does nothing.
+  - **test bit**:
+    - set (1): immediately mute channel
+      - if the channel is a source of ring mod and/or hard sync, those stop working until the bit is reset.
```

Comment:
Hmmm....... are you sure?

Reply:
Yes, I tested it yesterday and ring mod/hard sync stop working. I used ch1 as source and 2nd channel stopped being modulated or synced when I set test bit on channel 1.

Reply:
GOt it.